src/FormParseNodeFactory.php

```diff
@@ -35,6 +35,7 @@ public function getRootParseNode(string $contentType, StreamInterface $rawRespon
                 $finalResult[$key] []= $field[1];
             }
             $fields2 = array_map(fn ($item) => count($item) > 1 ? $item : $item[0], $finalResult);
+            var_dump($fields2);
```

Comment:
```suggestion
            var_dump($fields2);
```
Is this intentional?